components/automate-ui/src/app/page-components/projects-filter-dropdown/projects-filter-dropdown.component.ts

```diff
@@ -33,27 +37,36 @@ export class ProjectsFilterDropdownComponent {
   // so they can be filtered while maintaining the actual options.
   filteredOptions: ProjectsFilterOption[] = [];
 
-  filterValue = '';
-
   optionsEdited = false;
 
   dropdownActive = false;
 
+
+  ngOnInit() {
+    this.projectsFilterService.filterValue$.subscribe((filterValue) => {
```

Comment:
```suggestion
    this.projectsFilterService.filterValue$.subscribe(filterValue => {
```

Reply:
Thanks for taking the time to answer this so thoroughly.  While I suspect we'll end up using my other PR - this was my first time using a subscription and it was a good introduction.  It makes sense that we would want to unsubscribe as soon as we don't need that subscription anymore and I'll keep that tight coupling in mind moving forward.